ci/setup-build-test-gdk.sh

```diff
@@ -7,6 +7,27 @@ fi
 
 source /opt/improbable/environment
 
+# Download the unreal-engine.version file from the GDK repo so we can run the example project builds on the same versions the GDK was run against.
+# This is not the pinnacle of engineering, as we rely on GitHub's web interface to download the file, but it seems like GitHub disallows git archive
+# which would be our other option for downloading a single file.
+# Also resolve the GDK branch to run against. The order of priority is:
+# TEST_REPO_BRANCH envvar > same-name branch as the branch we are currently on > UnrealGDKVersion.txt > "master".
+GDK_BRANCH_LOCAL="${TEST_REPO_BRANCH:-}"
+if [ -z "${GDK_BRANCH_LOCAL}" ]; then
+    GDK_REPO_HEADS=$(git ls-remote --heads "git@github.com:spatialos/UnrealGDK.git" "${BUILDKITE_BRANCH}")
```

Comment:
```suggestion
    TEST_REPO_HEADS=$(git ls-remote --heads "${TEST_REPO_URL}" "${BUILDKITE_BRANCH}")
```
TEST_REPO_URL will have to be moved, or this branch detection code will have to be moved.